prettypandas/styler.py

```diff
@@ -46,8 +47,15 @@ def apply_pretty_globals():
 
 
 Formatter = namedtuple("Formatter", "subset, function")
-
-
+remove_nan_formatter = lambda x: '' if x == np.nan else x
+
+def PrettyPandasNoIndex(dataframe, table_styles = [], *args, **kwargs):
+    return PrettyPandas(dataframe, 
+                        *args, 
+                        table_styles = table_styles + [{'selector': '.row_heading', 'props': [('display', 'none')]}, 
+                                                       {'selector': '.blank', 'props': [('display', 'none')]}],
+                        **kwargs)
+                        
```

Comment:
Yep, sorry this was a quick dirty hack with emphasis on the dirty.

How about I merge it into the PrettyPandas class as an option kwarg e.g. hide_index=False?